templates/shortcode-donation-grid.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * This template is used to display the donation grid with [donation_grid]
+ */
+
+// Exit if accessed directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+$form_ID       = get_the_ID(); // Form ID.
+$give_settings = $args[0]; // Give settings.
+$atts          = $args[1]; // Shortcode attributes.
+?>
+
+<div class="give-donation-grid-item give-grid-col-<?php echo esc_attr( $atts['columns'] ); ?>">
+	<div class="box">
+		<?php
+		// The featured image.
+		if ( 'enabled' === $give_settings['form_featured_img'] && has_post_thumbnail() ) {
+			printf( '<div class="donation-grid-featured-image">' );
+			the_post_thumbnail();
+			printf( '</div>' );
+		}
+
+		// The Form title.
+		the_title( '<span class="donation-grid-title">', '</span>' );
+
+		// The progess bar for goal.
+		if ( 'enabled' === get_post_meta( $form_ID, '_give_goal_option', true ) && 'true' === $atts['show_goal'] ) {
+			echo '<div class="grid-item-progress">';
+			give_show_goal_progress( $form_ID );
+			echo '</div>';
+		}
+
+		// The excerpt.
+		if ( 'enabled' === $give_settings['forms_excerpt'] && 'true' === $atts['show_excerpt'] ) {
+			printf( '<div class="donor-grid-excerpt">%s</div>', get_the_excerpt() );
+		}
+
+		// The 'Donate Now' button.
+		if ( 'redirect' === $atts['display_type'] && 'redirect' === $atts['display_type'] ) {
+
+			// 'Donate Now' button if the 'display_type' attribute is set to 'redirect'
+			printf( '<a class="grid-donate-now" href="%1$s">%2$s</a>', get_the_permalink(), apply_filters( 'donation_grid_donate_now', esc_html__( 'Donate Now', 'grid' ) ) );
+
+		} elseif( 'modal' === $atts['display_type'] ) {
+
+			// 'Donate Now' button if the 'display_type' attribute is set to 'modal'
+			printf( '<a class="grid-donate-now grid-donate-now-modal-button" data-effect="mfp-zoom-out" href="#popup-form-%1$s">%2$s</a>', get_the_ID(), apply_filters( 'donation_grid_donate_now', esc_html__( 'Donate Now', 'grid' ) ) );
```

Comment:
@Sidsector9 can you make the effect a slide down/up to match how to works with the "Modal" Display Mode? You'll see it slides in and out sort of like bootstrap modals.

Reply:
@DevinWalker Updated the animation